subsystems/mvdm/ntvdm/dos/dos32krnl/country.c

```diff
@@ -212,7 +212,7 @@ WORD DosIfCharYesNo(WORD Char)
     if (Char == YesNoTable[0])
         return 0x0001;
     /* Unknown type */
-        return 0x0002;
+    return 0x0002;
```

Comment:
What's the warning corresponding to this?

Reply:
I presume it would be GCC's -Wmisleading-indentation since it looks like it was indented like it was a part of the if statement

Reply:
Correct, misleading indentation warning.